nixos/modules/services/monitoring/glpi-agent.nix

```diff
@@ -0,0 +1,118 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.glpiAgent;
+
+  configFile = pkgs.writeText "agent.cfg" ''
+    # Server configuration
+    server = ${lib.concatStringsSep ", " cfg.servers}
+
+    # Configuration
+    ${cfg.extraConfig}
+  '';
+
+in
+{
+  options = {
+    services.glpiAgent = {
+      enable = lib.mkEnableOption "GLPI Agent";
+
+      package = lib.mkPackageOption pkgs "glpi-agent" {
+      };
+
+      servers = lib.mkOption {
+        type = lib.types.listOf lib.types.str;
+        description = ''
+          The URLs of the GLPI servers to connect to.
+        '';
+        example = [ "https://glpi.example.com/inventory" ];
```

Comment:
You can drop this if you make the whole thing an RFC42-style attrset.

Reply:
Hi,

Why ? i think example is good information.
You have more information about this ?

Best Regards

Reply:
I mean the option itself, not just the example.